active/PipsPager/PipsPager.md

```diff
@@ -0,0 +1,209 @@
+In many applications, media and data is shown in various layouts like lists, grids, and tables. Instead of scrolling or panning through the content, users are able to page through the content with previous and next buttons or jump to specific pages. This UI pagination behavior is supported in WinUI currently with the PagerControl. See PagerControl [spec](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/user/savoyschuler/pagercontrol/active/PagerControl/PagerControl.md) and PagerControl [proposal](https://github.com/microsoft/microsoft-ui-xaml/issues/60) for the latest updates.
+
+A common UI pattern not yet supported by the existing PagerControl visualizations appears in media viewer scenarios like photo carousels or app lists. In these scenarios, the pages are commonly indicated with 'dot' glyphs instead of numbers. Conceptually, this is similar to an unordered list represented by bullet points instead of numbers. We would like to introduce a PipsPager to be used when the page content is not explicitly ordered and a symbol or glyph can instead represent the page number. 
+
+# Description
+The PipsPager is a UI component that provides a standard interaction for pagination with a layout view (ListView, GridView, ItemsRepeater, DataGrid, etc.) when the page numbers do not need to be visually known. The PipsPager, like the other pager controls, is independent of the layout views and does not control any of the data being displayed in the layout view.
+
+## Is this the right control?
+Use a PipsPager when the content in the layout is not explicitly ordered by relevancy or you desire a glyph-based representation of numbered pages. This UI is commonly used in photo viewers, app lists, and when display space is limited. The PipsPager can be oriented vertically or horizontally.
+
+### What is a pip?
+Pips represent a unit of numerical value, typically represented as dots, however can be customized to other glyphs like dashes or squares. In the PipsPager control by default, a solid dot will represent each page of the layout. They can be selected by the user to navigate to a specific page.
+
+# Examples
+
+## Create a PipsPager
+A PipsPager with five pages that can be selected by the user. By default, the previous and next buttons are collapsed and the pips are oriented horizontally.
+
+<!-- ![A sketch of five horizontal dots with the first selected.](images/PipsPager_HorizontalCollapsed.jpg) -->
+
+XAML
+```XAML 
+    <muxc:PipsPager NumberOfPages="5"/>
+```
+
+## PipsPager with visible buttons
+The visibility of the previous and next buttons can be set via the PreviousButtonVisibility and NextButtonVisibility properties. These buttons allow the user to select a page
+- Visible: The button is visible and selectable. An exception to this is that if the selected page is on the associated "edge" of the button then the button will be hidden.
+  - The edge for the previous button is the first page and the edge for the next button is the last page.
```

Comment:
So if you click on the center of the button it goes to the next/prev page, but if you click on the _edge_ of the button it jumps to the last/first page? How close to the edge do you have to click?

Reply:
@MikeHillberg edge in this context doesn't literally mean "edge of the button" but rather "edge" of the pager. It means that if we're on the first page - the previous button will be hidden, alternatively if we are on the last page - the last page will be hidden. This is due to the reason, that we cannot go any further back or forward :)

Reply:
I was going to comment here that I think we need new terminology for this concept as its commonly confused, what makes more sense to you @MikeHillberg?

Reply:
I see. In scrolling this is roughly what we call the "extent"?

Something like:

> Visible: The button is visible and enabled, except it is hidden at the extents. For example, when the current page is the first page, the previous button is hidden.

(Also changed 'selectable' to 'enabled', because I don't think this is a selection concept.)

Reply:
Updating to use that phrasing, thanks!